frame/claims/src/lib.rs

```diff
@@ -352,7 +351,8 @@ impl<T: Config> Module<T> {
 
 	fn pot<C: LockableCurrency<T::AccountId>>() -> C::Balance {
 		// Already lock minimal balance in the account, no need to worry about to be 0.
-		C::usable_balance(&Self::account_id())
```

Comment:
Looks like the `usable_balance` is only used for balance rpc?

Reply:
Almost.
And I think we can completely remove this later.